index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Chimwemwe's Portfolio</title>
+    
+    <link rel="stylesheet" href="styles.css">
+
+</head>
+<body>
+<header>
+    <nav class="nav-bar">
+
+    <div class="logo">
+        <a href="#">My logo</a>
+    </div>
+        <div class="menu-icon"> <img src="images/menu.png" alt="menu navbar"></div>
+    </nav>
+</header>
+
+    <section class="first-section" id="section-one">
+        <div class="text">
+            <h1>
+                I'm Chimwemwe glad to see you!
```

Comment:
- [Optional] Please notice the content Should be  `I'm Chimwemwe Glad to see you!` with `Glad` having an uppercase `G`, but on yours, it's a small `g`, It helps to be keen on details, especially in future jobs. :+1: 
- [Optional] Please also structure the line break for the content, so that it looks like the one on the Figma

##### Your design
![Screenshot from 2022-05-27 07-26-45](https://user-images.githubusercontent.com/58771507/170643368-cd741604-8d6c-4498-b93c-f75ac2b330e7.png)

##### Figma design
![Screenshot from 2022-05-27 06-46-53](https://user-images.githubusercontent.com/58771507/170643562-5afb773e-9066-4e3b-8408-979050c697fa.png)